src/bidiMapper/modules/network/NetworkProcessor.ts

```diff
@@ -106,11 +106,7 @@ export class NetworkProcessor {
         postData: getCdpBodyFromBiDiBytesValue(body),
       });
     } catch (error) {
-      // https://source.chromium.org/chromium/chromium/src/+/main:content/browser/devtools/protocol/fetch_handler.cc;l=169
-      if ((error as any)?.message.includes('Invalid header')) {
-        throw new InvalidArgumentException('Tried setting invalid header');
-      }
-      throw error;
+      NetworkProcessor.wrapInterceptionError(error);
```

Comment:
I would still be in favor of the helper function just wrapping exception, but throwing it from this place, not the wrapper function. Pure for readability.